lib/test_prof/before_all/adapters/active_record.rb

```diff
@@ -6,17 +6,27 @@ module Adapters
       # ActiveRecord adapter for `before_all`
       module ActiveRecord
         class << self
+          def all_connection_classes
+            @all_connection_classes ||= [::ActiveRecord::Base] + ::ActiveRecord::Base.descendants.select do |descendant|
```

Comment:
Maybe, we can use the AR configuration here? Similarly to what Rails does for fixtures: https://github.com/rails/rails/blob/fa09e3c6787d8dd5f3eff30e07deb67b249af6b6/activerecord/lib/active_record/test_fixtures.rb#L110-L114